src/Libraries/Microsoft.Extensions.Http.Resilience/Microsoft.Extensions.Http.Resilience.csproj

```diff
@@ -38,4 +38,29 @@
     <InternalsVisibleToTest Include="$(AssemblyName).Tests" />
     <InternalsVisibleToDynamicProxyGenAssembly2 Include="*" />
   </ItemGroup>
+
+  <ItemGroup>
+    <None Include="buildTransitive\$(MSBuildProjectName).targets"
+          CopyToOutputDirectory="PreserveNewest"
+          Pack="true"
+          PackagePath="buildTransitive\$(MinimumSupportedTfmForPackaging)\" />
+    
+    <!-- For net462 we have to rename the .targets file, because our build process automatically adds a .targets file
+         with the same name as the project file. Then we'll import the renamed .targets file in the automatically
+         added .targets file. For that we need to set the _AdditionalNETStandardCompatErrorFileContents variable. -->
+    <None Include="buildTransitive\$(MSBuildProjectName).targets"
+          CopyToOutputDirectory="PreserveNewest"
+          Pack="true"
+          PackagePath="buildTransitive\net462\Microsoft.Extensions.Http.Resilience.net462.targets" />
```

Comment:
Shouldn't this be conditional?

Reply:
I believe it shouldn't. We just want to copy the .targets file into the buildTransitive folder. I tested it and it worked. Do you have some kind of condition in mind?

We have similar tag in another file and it doesn't use any condition:
https://github.com/dotnet/extensions/blob/3131920ec0aa445af3d8bc615a088b5caf81fac4/src/Libraries/Microsoft.Extensions.Telemetry.Abstractions/Microsoft.Extensions.Telemetry.Abstractions.csproj#L27

Reply:
In the telemetry we aren't doing anything fancy. And if you exclude `net462` from TFMs, the telemetry package will only package `buildTransitive\net6.0`. But in this package, it will package `buildTransitive\net462` regardless, won't it?

Reply:
Added a condition to check whether we need to pack buildTransitive for net462. The condition is the same that is used [here](https://github.com/dotnet/extensions/blob/dc1ba70804189b12ada17bd8b3ac56be93ebdf66/eng/MSBuild/Packaging.targets#L20C29-L22C169).